src/main/java/net/cytonic/cynturion/auditlog/Category.java

```diff
@@ -0,0 +1,39 @@
+package net.cytonic.cynturion.auditlog;
+
+/**
+ * An enum for the different audit log categories
+ */
+public enum Category {
+    /**
+     * A player ban
+     */
+    BAN,
+    /**
+     * A player unban
+     */
+    UNBAN,
+    /**
+     * A player mute
+     */
+    MUTE,
+    /**
+     * A player unmute
+     */
+    UNMUTE,
+    /**
+     * A player IP ban
+     */
+    IPBAN,
+    /**
+     * A player IP unban
+     */
+    IPUNBAN,
+    /**
+     * A player kick
+     */
+    KICK,
+    /**
+     * A player warn
+     */
+    WARN
+}
```

Comment:
This needs to be moved to commons, since we now have it in two places. (Make the code DRY, yk)